web-crawler/src/main/java/com/google/sps/webcrawler/WebCrawler.java

```diff
@@ -199,23 +220,32 @@ private boolean waitForCrawlDelay(long timeToDelay) {
   /** 
    * Stores {@code NewsArticle}'s metadata and content into the database, following a predesigned
    * database schema.
-   * Requires "gcloud config set project project-ID" to be set correctly. Assumes "content" to be
-   * occupy fewer than 1500 bytes.
+   * Requires "gcloud config set project project-ID" to be set correctly. Data beyond 1500 bytes
+   * be excluded from indexes.
```

Comment:
Are NewsArticles with more than 1500 bytes of content dropped completely, or is the content just truncated to 1500 bytes?

Reply:
`NewsArticle` and `content` are not truncated. It's just that the `content` is not indexed in the database (it also doesn't need to be indexed anyways). Sorry about the confusion. I'll rewrite the comment to make this clear